doc/TEMPLATES.md

```diff
@@ -291,6 +291,49 @@ html`<div>${fragment}</div>`;
 // Error: Unexpected child element count of zero for given DocumentFragment.
 ```
 
+## Supported native tags
+
+The default template engine is opinionated about which elements are allowed in
+order to reduce complexity and improve performance. The following tags are
+supported:
+
+```html
```

Comment:
Thought it would be nice to present this with `html` since that’s what it’s about. Open to different formatting options, but this felt like it would get the point across in a concise way.